src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSearchService.kt

```diff
@@ -61,6 +63,7 @@ class PrisonerSearchService(
     val query = queryBuilder(searchCriteria)
     return query?.let {
       val searchSourceBuilder = SearchSourceBuilder().apply {
+        size(RESULT_HITS_MAX)
```

Comment:
In the two `queryBy` methods it looks like there's a few lines of duplicated code (up to building the `searchRequest`). Could we extract out the duplication?

Reply:
There is a difference in the lines of code so I didn't think it was worth it